modules/core/src/main/scala/org/tessellation/http/routes/GlobalSnapshotRoutes.scala

```diff
@@ -17,6 +20,12 @@ final case class GlobalSnapshotRoutes[F[_]: Async: KryoSerializer](globalSnapsho
   private val prefixPath = "/global-snapshot"
 
   private val httpRoutes: HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "latest" / "ordinal" =>
+      globalSnapshotStorage.head.map(_.map(_.ordinal)).flatMap {
+        case Some(ordinal) => Ok(ordinal)(Applicative[F], circeEntityEncoder)
```

Comment:
hm, I'd import encoder selectively for that case:
```
case GET -> Root / "latest" / "ordinal" => {
  import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
  
  globalSnapshotStorage... Ok(ordinal) | NotFound()
}

case GET -> Root / "latest" => {
  import org.tessellation.ext.codecs.BinaryCodec._
  
  globalSnapshotStorage.head... Ok(snapshot)
}
```

what do you think guys? @TheMMaciek @kpudlik @tbekas @kriscieslak

Reply:
I like the idea. Gives more control and visibility over the implicits.